oci/oci-how-to/deploy-pro-container-on-pro-kubernetes-cluster.rst

```diff
@@ -0,0 +1,110 @@
+Deploy Ubuntu Pro container image to Pro Kubernetes cluster
+===========================================================
+
+You can easily deploy your Ubuntu Pro container images to Ubuntu Pro Kubernetes clusters.
+The Kubernetes cluster needs to be on Ubuntu Pro nodes. Read more on the
+`Ubuntu Pro <https://ubuntu.com/pro>`_ website.
+
+
+Prerequisites
+-------------
+
+You will need --
+
+- an Ubuntu Pro container image on a private registry. See how to build one
+  :doc:`here <build-ubuntu-pro-container-image>`.
+  The image should be kept private to avoid sharing your Ubuntu Pro subscription with
+  unwanted users since the image is linked to your Ubuntu Pro subscription.
+  
+  You can publish the image to your private registry. First, tag the image with your
+  private registry details using:
+
+  ::
+
+		docker tag <pro-image-name> <private-registry-pro-image-name>
+	
+  Finally, publish the image with:
+  
+  ::
+
+		docker push <private-registry-pro-image-name>
+
+- a Kubernetes cluster with a Ubuntu Pro subscription. See the cloud-specific guides to
```

Comment:
I think this hints to what I'm suggesting in https://github.com/canonical/ubuntu-cloud-docs/pull/33#pullrequestreview-1388568279.

basically, the prerequisite is a pro cluster. So as you are doing here, you can create a small section to quickly explain how that works for multiple clouds, either pointing to existing documentation or mentioning what you already have in the other open PRs

Reply:
Added a section with instructions on how to deploy Pro K8s clusters.